src/openms/include/OpenMS/ANALYSIS/ID/IDScoreGetterSetter.h

```diff
@@ -290,7 +290,7 @@ namespace OpenMS
                     std::vector<IDType> &ids,
                     const std::string &score_type,
                     bool higher_better,
-                    Args &... args)
+                    Args &&... args)
```

Comment:
looks good to me... not sure if there is a better solution to having higher_better unused in one of the members

Reply:
It probably should make an if case as well